### DISCUSSION TITLE
display logic depth and logic path

### DISCUSSION THREAD

**Discussion by shengqilong** [YosysHQ/yosys#3710] [General]
How to get the maximum logic depth and longest logic path after synthesis ？

**Comment by Ravenslofty** [comment]
`ltp` is **l**ongest **t**opological **p**ath of logic by depth; `sta` is **s**tatic **t**iming **a**nalysis (although one would first need to re-read the simulation library to include the timing arcs)

**Reply by shengqilong** [reply]
The length displayed by ltp is very different from the length displayed after cell mapping，If cmos_cells.lib do not contain any timing information sta will not display the logical depth
